config/default.js

```diff
@@ -1,9 +1,21 @@
+const PostGraphileConnectionFilterPlugin = require('postgraphile-plugin-connection-filter');
+// const path = require('path');
+
 module.exports = {
   app: {
     port: process.env.PORT || 3000,
     suppressErrorStack: process.env.SUPPRESS_ERROR_STACK || false,
   },
 
+  db: {
+    user: 'plyo_mailer',
```

Comment:
```suggestion
    user: 'plyo_boilerplate',
```

would be nice to have abstract name here, so developer must to change it to make code working

Reply:
@jifeon uh, accidentally added it